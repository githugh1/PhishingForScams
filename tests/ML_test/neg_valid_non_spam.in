{
  "email_subject": "Important Business Proposal",
  "email_body": "Dear Sir/Madam, I have a lucrative business...",
  "sender": "business@email.com"
}
