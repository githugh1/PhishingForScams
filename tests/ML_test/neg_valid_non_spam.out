{
  "is_spam": false,
  "confidence": 0.2,
  "message": "This email is not considered spam."
}
